doc/gen_book.rs

```diff
@@ -324,6 +326,8 @@ impl SubCommandPages {
 
         for board in &board_info {
             let mut env = Environment::new();
+            env.add_filter("slugify", slugify::<String>);
+            env.add_filter("capitalize2", capitalize);
```

Comment:
Sounds a bit more descriptive:
```suggestion
            env.add_filter("capitalize_first", capitalize);
```